.github/ISSUE_TEMPLATE/backlog_task.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Backlog Task
```

Comment:
How do we want to use this relative to the Design/Research and DevEx/OpEx templates? (design_research.md and devex_opex.md) Does this replace the need for those?

Reply:
That's a good question.  Those are perhaps specialized versions of this template?  Or maybe we just remove them?